packages/amplify-graphql-model-transformer/src/resolvers/query.ts

```diff
@@ -16,26 +16,62 @@ import {
   equals,
   bool,
   and,
+  isNullOrEmpty,
+  list,
+  forEach,
+  nul,
 } from 'graphql-mapping-template';
 
+const authFilter = methodCall(ref('ctx.stash.get'), str('authFilter'));
+
 /**
  * Generate get query resolver template
  */
 export const generateGetRequestTemplate = (): string => {
   const statements: Expression[] = [
-    set(ref('GetRequest'), obj({ version: str('2018-05-29'), operation: str('GetItem') })),
+    set(ref('GetRequest'), obj({ version: str('2018-05-29'), operation: str('Query') })),
```

Comment:
I know we talked about this offline, but what was the specific reason for needing to switch the operation again?

Reply:
The reason for this is so we can apply an auth filter as opposed to running auth resolver logic after the get.